### ISSUE TITLE
[Programmatic Access – Make Code – Home Page] Role is defined for all tiles present under ‘Home Page’

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3439]
**User Experience:**
User would find difficulty to identify the controls if role is not defined for that control as user won't understand the purpose of that control and also he/she would surely miss the functionality over that control.

**Note**: User credentials should NOT be included in the bug. 

**Pre-Requisite:**
Open F12 Developer tool.

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/#
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify that role is defined for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.)

**Actual Result:**
Role is not defined for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.) under ‘Home page’ and screen reader is reading as group and due to this user won't be able to identify the control and won't be perform functionality over that control. 

**Expected Result:**
Role should be define as 'Button' for tiles control like(Flashing Heart, Name tag, Smiley Buttons, Dice, Love Meter, Micro Chat, Coin Toss, Step counter, Snap the dot, Coin flipper, 7 second etc.) under ‘Home page’, so that user could easily identify the control and take an action accordingly.

**Note:**
Same issue exists on Firefox + NVDA & Chrome + JAWS.

**MAS Reference:**
[MAS 4.1.2 – Name, Role, Value ](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Reference Links:**
•	[Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA
#URL:https://makecode.microbit.org/#
